src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/CodeAnalysisPropertyPage.xaml

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="CodeAnalysisPropertyPage"
+      Description="Settings"
+      DisplayName="Code Analysis"
+      PageTemplate="generic"
+      Order="500"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="AllAnalyzers"
+              DisplayName="All analyzers" />
+
+    <Category Name="NetAnalyzers"
+              DisplayName=".NET analyzers" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <BoolProperty Name="RunAnalyzersDuringBuild"
+                DisplayName="Run on build"
+                Description="Always run code analysis on build."
+                HelpUrl="https://docs.microsoft.com/visualstudio/code-quality/disable-code-analysis"
```

Comment:
Is it possible to make these URLs go via the fwlink redirection service (https://github.com/dotnet/project-system/issues/7262)?
Thanks for removing the culture (i.e. en-us) in it 😄

Reply:
Just saw you opened an issue to address this, so closing it in favor of it.